src/main/java/org/aion/wallet/account/AccountManager.java

```diff
@@ -203,20 +202,20 @@ private AccountDTO addExternalAccount(final ECKey key, final byte[] fileContent,
         if (accountDTO == null) {
             throw new ValidationException("Failed to create account");
         }
-        processAccountAdded(accountDTO, fileContent, false);
+        processAccountAdded(accountDTO, fileContent);
         return accountDTO;
     }
 
     public SortedSet<TransactionDTO> getTransactions(final String address) {
         return addressToTransactions.getOrDefault(address, Collections.emptySortedSet());
```

Comment:
I think this should be an UnmodifiableSet and have special assessors because it's hard to maintain all the places where we write to the set.